oauth2-authorization-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationServerConfigurer.java

```diff
@@ -196,7 +196,9 @@ public void init(B builder) {
 		if (exceptionHandling != null) {
 			LinkedHashMap<RequestMatcher, AuthenticationEntryPoint> entryPoints = new LinkedHashMap<>();
 			entryPoints.put(
-					new OrRequestMatcher(this.tokenEndpointMatcher, this.tokenRevocationEndpointMatcher),
+					new OrRequestMatcher(
```

Comment:
This change leaves `this.tokenEndpointMatcher` and `this.tokenRevocationEndpointMatcher` in an inconsistent state. The members should be updated.

Reply:
@jgrandja I know but all other matchers are also in an inconsistent state. I can change all matchers if you agree.

Reply:
Yes, please change all matchers and anything else that applies within `OAuth2AuthorizationServerConfigurer`.

For the tests, take a look at the existing integration tests in package `org.springframework.security.config.annotation.web.configurers.oauth2.server.authorization`

Reply:
I will try to change it before the end of the week.